src/main/scala/org/phenoscape/kb/KB.scala

```diff
@@ -189,8 +190,44 @@ OPTIONAL {
     App.executeSPARQLQueryString(query.text, res => Instant.parse(res.getLiteral("date").getLexicalForm)).map(_.head)
   }
 
+  def getKBMetadata: Future[KBMetadata]= {
+    val builtFut = buildDate
+    val ontologiesFut = kbOntologies
+    for {
+      built <- builtFut
+      ontologies <- ontologiesFut
+    } yield KBMetadata(built, ontologies)
+  }
+
+  def kbOntologies: Future[Seq[(IRI, IRI)]] = {
+    val query = sparql"""
+    PREFIX owl: <http://www.w3.org/2002/07/owl#>
+    SELECT ?ont ?version
+    FROM $KBMainGraph
+    WHERE {
+      ?ont $rdfType owl:Ontology .
+  	  ?ont owl:versionIRI ?version
+    }
+  """
+
+    App.executeSPARQLQueryString(query.text, qs => (IRI.create(qs.getResource("ont").getURI), IRI.create(qs.getResource("version").getURI)))
+  }
 }
 
+case class KBMetadata(built: Instant, ontologies: Seq[(IRI, IRI)]) extends JSONResultItem {
+
+  def toJSON: JsObject = Map(
+    "build_time" -> built.toString.toJson,
+    "ontologies" -> ontologies.map{
```

Comment:
Even though I am suggesting to use `Set` in the constructor, I would then convert to Seq here and sort somehow before converting to JSON, to ensure consistent output.